lib/liulo/events/question.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Liulo.Events.Question do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+
+  schema "question" do
+    field :description, :string
+    field :is_anonymous, :boolean, default: false
+    field :status, QuestionStatusEnum, default: :pending
+    field :vote_count, :integer , default: 0
+    # field :owner_id, :id
+    # field :topic_id, :id
+    belongs_to(:owner, Liulo.Accounts.User)
+    belongs_to(:topic, Liulo.Events.Topic)
+
+    timestamps()
+  end
+  @required_fields ~w(description is_anonymous)a
```

Comment:
`is_anonymous` should be optional because it already have default value and in requirement user doesn't need to fill it